Cargo.toml

```diff
@@ -484,69 +186,16 @@ fsmc_lcd = ["dep:display-interface", "dep:display-interface-04"]
 ## SDIO peripheral support. See [sdio-host](https://crates.io/crates/sdio-host)
 sdio-host = ["dep:sdio-host"]
 
-dfsdm = []
-sai = []
-
-adc2 = []
-adc3 = []
-aes = []
-can1 = []
-can2 = []
-can3 = []
-cryp = []
-dac = []
-dcmi = []
-dfsdm1 = ["dfsdm"]
-dfsdm2 = ["dfsdm"]
-dma2d = []
-dsihost = []
-eth = []
-fmc = []
-fsmc = []
-fmpi2c1 = []
-gpiod = []
-gpioe = []
-gpiof = []
-gpiog = []
-gpioi = []
-gpioj = []
-gpiok = []
-i2c3 = []
-lptim1 = []
-ltdc = ["dep:micromath"]
-quadspi = []
-otg-fs = []
-otg-hs = []
-rng = []
-sai1 = ["sai"]
-sai2 = ["sai"]
-sdio = []
-spi3 = []
-spi4 = []
-spi5 = []
-spi6 = []
-spdifrx = []
-tim1 = []
-tim2 = []
-tim3 = []
-tim4 = []
-tim5 = []
-tim6 = []
-tim7 = []
-tim8 = []
-tim9 = []
-tim10 = []
-tim11 = []
-tim12 = []
-tim13 = []
-tim14 = []
-usart3 = []
-uart4 = []
-uart5 = []
-uart7 = []
-uart8 = []
-uart9 = []
-uart10 = []
+# Private features for testing only
+_pac_fsmc = []
```

Comment:
@therealprof 
I could not find any other solution for examples except fictive features.